### ISSUE TITLE
Brand new app can't launch due to improper use of `electron-is-dev`

### ISSUE THREAD

**Issue by jagthedrummer** [adopted-ember-addons/ember-electron#1693]
I just started a brand new app using `ember-cli@5.7.0`, and after installing `ember-electron` the app won't start. It complains about `electron-is-dev` being `require`d instead of `import`ed.

```
$ ember --version
ember-cli: 5.7.0
node: 18.19.1
os: darwin arm64
```

```
$ ember new ember-electron-test-five-point-seven
...
$ cd ember-electron-test-five-point-seven
$ ember install ember-electron
...
$ ember g ember-electron
...
$ ember electron
Checking dependencies in electron-app...
Environment: development

Build successful (5379ms)

Slowest Nodes (totalTime >= 5%)                                                                                        | Total (avg)
-----------------------------------------------------------------------------------------------------------------------+----------------------------------------
Babel: @ember/test-helpers (1)                                                                                         | 1802ms
ember-auto-import-webpack (1)                                                                                          | 815ms
Babel: ember-data (2)                                                                                                  | 379ms (189 ms)

Starting Electron...
App threw an error during load
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/node_modules/electron-is-dev/index.js from /Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/src/index.js not supported.
Instead change the require of /Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/node_modules/electron-is-dev/index.js in /Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/src/index.js to a dynamic import() which is available in all CommonJS modules.
    at c._load (node:electron/js2c/node_init:2:13672)
    at Object.<anonymous> (/Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/src/index.js:9:15)
    at c._load (node:electron/js2c/node_init:2:13672)
    at async loadApplicationPackage (file:///Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js:129:9)
    at async file:///Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js:241:9
```

**Comment by jagthedrummer** [comment]
I changed the version number in `electron-app/package.json` from `3.0.1` to `2.0.0`.

Before:
```
"electron-is-dev": "^3.0.1"
```

After:
```
"electron-is-dev": "^2.0.0"
```

Now the app will launch, but it complains about `devtron`:

```
$ ember electron
Checking dependencies in electron-app...
Environment: development

Build successful (2285ms)

Slowest Nodes (totalTime >= 5%)                                                | Total (avg)
-------------------------------------------------------------------------------+--------------------
ember-auto-import-webpack (1)                                                  | 774ms
Babel: @ember/test-helpers (1)                                                 | 330ms
Babel: ember-data (2)                                                          | 287ms (143 ms)

Starting Electron...
Installing Devtron from /Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/node_modules/devtron
Failed to install Devtron:  TypeError: electron.BrowserWindow.addDevToolsExtension is not a function
    at exports.install (/Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/node_modules/devtron/api.js:13:35)
    at App.<anonymous> (/Users/jgreen/projects/ember-electron-test-five-point-seven/electron-app/src/index.js:37:26)
    at App.emit (node:events:514:28)
(node:29564) ExtensionLoadWarning: Warnings loading extension at /Users/jgreen/Library/Application Support/ember-electron-test-five-point-seven/extensions/bmdblncegkenkacieihfhpjfppoconhi:
  Permission 'contextMenus' is unknown or URL pattern is malformed.

(Use `Electron --trace-warnings ...` to show where the warning was created)
[29564:0322/110848.021556:ERROR:extensions_browser_client.cc(68)] Extension Error:
  OTR:     false
  Level:   1
  Source:  manifest.json
  Message: Service worker registration failed. Status code: 15
  ID:      ebbipcpbaikgfkldfnoojibcbkonfkcj
  Type:    ManifestError
[29564:0322/110848.021626:ERROR:extensions_browser_client.cc(68)] Extension Error:
  OTR:     false
  Level:   2
  Source:  chrome-extension://ebbipcpbaikgfkldfnoojibcbkonfkcj/background.js
  Message: Uncaught TypeError: Cannot read properties of undefined (reading 'onClicked')
  ID:      ebbipcpbaikgfkldfnoojibcbkonfkcj
  Type:    RuntimeError
  Context: chrome-extension://ebbipcpbaikgfkldfnoojibcbkonfkcj/background.js
  Stack Trace:
    {
      Line:     93
      Column:   1
      URL:      chrome-extension://ebbipcpbaikgfkldfnoojibcbkonfkcj/background.js
      Function: (anonymous function)
    }
```

**Comment by bendemboski** [comment]
@jagthedrummer sorry, we need to cut a release -- see #1640

**Comment by bendemboski** [comment]
Fixed in v7.0.0
